lib/mosaic-network/src/main/java/org/eclipse/mosaic/lib/coupling/ClientServerChannel.java

```diff
@@ -378,6 +378,7 @@ public int writeRemoveNodesMessage(long time, List<Integer> ids) throws IOExcept
     }
 
     // @param channelId the channelID               //TODO:make enum from
```

Comment:
Is this TODO still relevant?

Reply:
I have no clue

Reply:
I just noticed this, since it was right above changes you made, so I thought maybe you looked at it 🤷

Reply:
I just re-formatted this file 😆